detectron/utils/vis.py

```diff
@@ -96,7 +96,6 @@ def get_class_string(class_index, score, dataset):
 
 def vis_mask(img, mask, col, alpha=0.4, show_border=True, border_thick=1):
     """Visualizes a single binary mask."""
-
```

Comment:
It would be nice to leave this as it was (new line without trailing spaces) as the change is unrelated to the fix.